### DISCUSSION TITLE
Directorylister cache

### DISCUSSION THREAD

**Discussion by iknowl97** [DirectoryLister/DirectoryLister#928] [Help & Support]
Hello,  can someone help me to disable cache growing ? 

Can I do it from /config/cache.php file ?

**Comment by PHLAK** [comment]
Starting with [v3.8.3](https://github.com/DirectoryLister/DirectoryLister/releases/tag/3.8.3) the cache lifetime has been updated to 1 hour and file-backed cache drivers will automatically prune expired items based on a request "lottery".

Which version of Directory Lister and what cache driver are you using?
